### PR TITLE
fix missing flag value bindings in kubectl

### PR DESCRIPTION
**Pull request by dixudx**
**What this PR does / why we need it**:
When working #63644, I find there are still some flag not bound correctly.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
xref #60366

**Special notes for your reviewer**:
/cc soltysh deads2k 
/cc kubernetes/sig-cli-api-reviews 
**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by dixudx** [comment]
/kind cleanup

@soltysh I think we should make sure all flags bound correctly for new subcommands in kubectl. I think we should add to [kubectl conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/kubectl-conventions.md). WDYT?

File: pkg/kubectl/cmd/get/get.go [hunk 1]
```diff
@@ -720,12 +718,12 @@ func (o *GetOptions) printGeneric(r *resource.Result) error {
 	return utilerrors.Reduce(utilerrors.Flatten(utilerrors.NewAggregate(errs)))
 }
 
-func addOpenAPIPrintColumnFlags(cmd *cobra.Command) {
-	cmd.Flags().Bool(useOpenAPIPrintColumnFlagLabel, false, "If true, use x-kubernetes-print-column metadata (if present) from the OpenAPI schema for displaying a resource.")
+func addOpenAPIPrintColumnFlags(cmd *cobra.Command, opt *GetOptions) {
```
Comment (deads2k) [inline pkg/kubectl/cmd/get/get.go]: Can you actually make this `func (o *GetOptions) AddFlags(flagset)` and put both bind bits in here?

**Comment by deads2k** [comment]
Great update. I won't let perfect be the enemy of good.  I would like to see some of these flag structs start growing their own `AddFlags` methods.

/lgtm

**Pull request merged**